src/transform.js

```diff
@@ -254,7 +255,7 @@ transform.transformCoordinates = function (
 
   var trans = transform({source: srcPrj, target: tgtPrj}), output;
   if (util.isObject(coordinates) && 'x' in coordinates && 'y' in coordinates) {
-    output = trans.forward({x: coordinates.x, y: coordinates.y, z: coordinates.z || 0});
+    output = trans.forward({x: +coordinates.x, y: +coordinates.y, z: +coordinates.z || 0});
```

Comment:
@manthey the reason I did not approve it earlier, as I had a question on the signs. I approve it now, but still curious. is that because of the notation that is now enforced?

Reply:
`+` casts a string to a float but does nothing to a float.  It is cheaper than `Number(<string or number>)`.

Reply:
A quick test shows the `+` takes 60% of the time of `Number`.